my_game.py

```diff
@@ -78,7 +78,9 @@ def __init__(self, center_x, center_y):
             filename="images/Meteors/meteorGrey_big1.png", 
             scale=SPRITE_SCALING
         )
-        
+
+        center_x = random.randint(0, SCREEN_WIDTH)
+        center_y = random.randint(0, SCREEN_HEIGHT)
```

Comment:
Please set self.center_x and self.center_y directly. That way, you don't need the following two lines.

Reply:
Got it